src-ghc/Pact/Interpreter.hs

```diff
@@ -138,7 +137,8 @@ evalExec runner evalEnv ParsedCode {..} = do
 
 -- | For pre-installing modules into state.
 initStateModules :: HashMap ModuleName (ModuleData Ref) -> EvalState
-initStateModules modules = set (evalRefs . rsLoadedModules) (fmap (,False) modules) def
+initStateModules modules =
+  set (evalRefs . rsQualifiedDeps) (foldMap allModuleExports modules) $ set (evalRefs . rsLoadedModules) (fmap (,False) modules) def
```

Comment:
`foldMap` accomplish here, for reference?

Reply:
it's same as `foldl (\acc m -> HM.union acc (allModuleExports m))`

We for all loaded modules, we need the transitive dependencies in scope.